src/payment/data/service.js

```diff
@@ -1,51 +1,13 @@
-import pick from 'lodash.pick';
-import { logError } from '@edx/frontend-logging';
+import { App } from '@edx/frontend-base';
 
-import { configureApiService as configureCybersourceApiService } from '../payment-methods/cybersource';
-import { configureApiService as configurePayPalApiService } from '../payment-methods/paypal';
-import { configureApiService as configureApplePayApiService } from '../payment-methods/apple-pay';
-import { applyConfiguration, handleRequestError } from '../../common/serviceUtils';
-import { camelCaseObject } from '../../common/utils';
+import handleRequestError from './handleRequestError';
+import { camelCaseObject } from './utils';
 import { ORDER_TYPES } from './constants';
 
-let config = {
-  ACCOUNTS_API_BASE_URL: null,
-  ECOMMERCE_BASE_URL: null,
-  ECOMMERCE_API_BASE_URL: null,
-  ECOMMERCE_RECEIPT_BASE_URL: null,
-  ENVIRONMENT: null,
-  LMS_BASE_URL: null,
-};
-
-let apiClient = null; // eslint-disable-line no-unused-vars
-
-export function configureApiService(newConfig, newApiClient) {
-  applyConfiguration(config, newConfig);
-  config = pick(newConfig, Object.keys(config));
-  apiClient = newApiClient;
-
-  configureCybersourceApiService(newConfig, apiClient);
-  configurePayPalApiService(newConfig, apiClient);
-  configureApplePayApiService(newConfig, apiClient);
-
-  // For every ajax response, check if the API has
-  // responded with a redirect value. If so, redirect.
-  /* istanbul ignore next */
-  apiClient.interceptors.response.use((response) => {
```

Comment:
Is this now incorporated into frontend-base? I recall this behavior being new and unique to payment

Reply:
Nevermind I see this below

Reply:
👍